packages/@glimmer/syntax/test/template-locals-test.ts

```diff
@@ -45,6 +52,7 @@ QUnit.test('it works', function (assert) {
     'global-value',
     'some',
     'someOther',
+    'ComponentYieldingContextual',
```

Comment:
```suggestion
    'ComponentYieldingContextual',
    'InsideHash',
    'inside-another-hash',
```

I *think* this is the order it should show up in ☝🏼